packages/lib/src/components/Ach/components/AchInput/AchInput.tsx

```diff
@@ -150,6 +144,21 @@ function AchInput(props: ACHInputProps) {
                             <div className={classNames(['adyen-checkout__fieldset', 'adyen-checkout__fieldset--ach'])}>
                                 {<div className="adyen-checkout__fieldset__title">{i18n.get('ach.bankAccount')}</div>}
 
+                                <Field classNameModifiers={['bankAccountType', 'no-borders']} name={'bankAccountType'} useLabelElement={false}>
+                                    <RadioGroup
+                                        name={'bankAccountType'}
+                                        value={data.bankAccountType}
+                                        items={[
+                                            { id: 'checking', name: i18n.get('ach.checking') },
+                                            { id: 'savings', name: i18n.get('ach.savings') }
+                                        ]}
+                                        classNameModifiers={['gender']}
```

Comment:
gender?  🤔 
Looks like the `RadioGroup` component does not use `classNameModifiers` .

Reply:
Oh yeah, I need to make that change. Also it's missing the clean up on some billing address types that were removed. 
The change has been made for V5, I'm just waiting to submit translations together with it.